SharedCode/Repository/DB/PokemonRepositoryLocal.cs

```diff
@@ -118,6 +117,54 @@ public async Task<Result<List<PokemonLocal>>> GetPokemonLocalListAsync()
             }
             return Result.Fail<List<PokemonLocal>>("Can't get info from db");
         }
+
+        public void StoreEvolutionChain(EvolutionChainResponse evolutionChain)
+        {
+            var evolutionChainString = Newtonsoft.Json.JsonConvert.SerializeObject(evolutionChain);
+            var localEvolutionChain = new EvolutionChainLocal { Id = evolutionChain.id, Chain = evolutionChainString };
+            DatabaseManager.StoreData(localEvolutionChain, Constants.EvolutionChainTable);
+        }
+
+        public Result<EvolutionChainResponse> GetEvolutionChainByIdFromLocal(int id)
```

Comment:
maybe you can declare this function in the IPokemonRepositoryLocal as you are getting data from DB

Reply:
Done.